### ISSUE TITLE
Warning when creating a datagrid

### ISSUE THREAD

**Issue by martinRenou** [bloomberg/ipydatagrid#118]
I get the following warning when creating a table:
```
/home/martin/miniconda3/envs/ipydatagrid/lib/python3.7/site-packages/pandas/io/json/_table_schema.py:82: UserWarning: Index name of 'index' is not round-trippable
  warnings.warn("Index name of 'index' is not round-trippable")
```

**Comment by gaborbernat** [comment]
I assume this is no longer the case given the issue was a year ago, and no reproducible example is provided. I'll close this, but if you come back with a reproducible example I am happy to reopen it.

**Comment by ibdafna** [comment]
@gaborbernat I think this is still happening, so will reopen for now. It's not a crucial issue but we should look at it when we have finished working on other, high priority items.

**Comment by gaborbernat** [comment]
Do you have a reproducible example of this?

**Comment by ibdafna** [comment]
Yeah, it happens in one of the examples in DataGrid.ipynb or Pandas.ipynb

**Comment by ibdafna** [comment]
Resolved via #260.
